make-util.js

```diff
@@ -17,7 +17,7 @@ var downloadPath = path.join(repoPath, '_download');
 // list of .NET culture names
 var cultureNames = ['cs', 'de', 'es', 'fr', 'it', 'ja', 'ko', 'pl', 'pt-BR', 'ru', 'tr', 'zh-Hans', 'zh-Hant'];
 
-var allowedTypescriptVersions = ['4.0.2', '5.1.6'];
+var allowedTypescriptVersions = ['4.0.2', '4.9.5', '5.1.6'];
```

Comment:
Is there a reason why we can't use 5.1.6?

Our preference is 4.0.2 or 5.1.6 only as we're trying to limit the number of typescript versions we support across tasks. Note: We're updating typescript to 5.1.6 as part of the Node20 updates.